packages/application-components/src/components/portals-container/portals-container.tsx

```diff
@@ -16,6 +18,7 @@ type TObservableElementDimensions = {
   height: number;
   width: number;
 };
+type TStackingLayer = { stackingLevel: number; indentationLevel: number };
```

Comment:
Nit: Maybe add a comment what each intends to do. Otherwise requires some information gathering in code.